lib/commands/instrument.js

```diff
@@ -100,20 +106,7 @@ exports.handler = function (argv) {
     ? './lib/instrumenters/istanbul'
     : './lib/instrumenters/noop'
 
-  const nyc = new NYC({
```

Comment:
It's a little weird that this refactor happens in this pull, seems like an improvement to picking and choosing parameters though.

Reply:
@JaKXz saw that I was manually adding yet another option to this constructor, asked if we could do this.